src/pages/BlueSolo/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "urls": {
+    "match": ["*://www.bluesolo.org/*"]
```

Comment:
Why not directly use `*://www.bluesolo.org/manga/*`

Reply:
Done ✅